assets/js/cartbox.modal.js

```diff
@@ -85,10 +85,11 @@
 		
 		price *= inputEl.val();
 		
-		var decimals = $cartItem.data('priceFormat').split('.').pop();
+		var decimalSeperator = $cartItem.data('priceFormat').indexOf('.') > -1 ? '.' : ',';
+		var decimals = $cartItem.data('priceFormat').split(decimalSeperator).pop();
```

Comment:
How about using the format returned from `currency()->getFormatter()`?

Reply:
Actually should be `currency()->getCurrency()->getFormat()`

Reply:
Not sure what you mean - we already get and pass the format as a data attribute?

Reply:
Ah, you're right! Never mind then 🤦